LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I would like something here to explain more explicitly what this applies too. (Even if it redirects you to the documentation.)